### PR TITLE
Ag/edit drop down

### PR DESCRIPTION
**Pull request by agallant121**
@holmesm8 @caachz @TravisBorgsmiller 

Added styling to the drop down menus. I have already pulled master into this branch and dealt with the merge conflicts so it should be good to go. No functionality changes on here, only styling.

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -26,17 +26,44 @@ h1 {
...
-    font-family: 'Alice', serif;
-    font-size: 18px;
+    font-family: Lucida Console, Courier, monospace;
+    font-size: 20px;
     font-weight: normal;
     line-height: 23px;
     margin: 0 0 23px 0;
     }
+
+select {
```
Comment (agallant121) [inline app/assets/stylesheets/application.scss:53]: Added edits to drop down menus for styling

File: app/views/dashboard/show.html.erb [hunk 2]
```diff
@@ -7,10 +7,24 @@
 		</h4>
 	</div>
 <% end %><br><br><br><br>
+<div class="center-align">
+
+	<!-- <a class="waves-effect waves-light btn-small #c5e1a5 light-green lighten-3 black-text">
```
Comment (agallant121) [inline app/views/dashboard/show.html.erb:12]: This has been moved to the location I think we will have it but it is commented out for now until we can implement the functionality

File: app/views/dashboard/show.html.erb [hunk 3]
```diff
@@ -45,12 +59,4 @@
 				<%= submit_tag('Shake It Up!', class: "button") %>
 			<% end %>
 		</a>
-
-		<center>
```
Comment (agallant121) [inline app/views/dashboard/show.html.erb:49]: This has just been moved further up the page.

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -26,17 +26,44 @@ h1 {
...
     }
+
 h4 {
     font-family: 'Playfair Display', serif;
     font-weight: 900;
     line-height: 46px;
     font-size: 43px;
     margin: 0 0 23px 0;
     }
+
```
Comment (agallant121) [inline app/assets/stylesheets/application.scss:37]: Changed font style and size

File: app/assets/stylesheets/application.scss [hunk 5]
```diff
@@ -26,17 +26,44 @@ h1 {
...
 p {
-    font-family: 'Alice', serif;
-    font-size: 18px;
+    font-family: Lucida Console, Courier, monospace;
+    font-size: 20px;
     font-weight: normal;
     line-height: 23px;
     margin: 0 0 23px 0;
     }
+
```
Comment (TravisBorgsmiller) [inline app/assets/stylesheets/application.scss:52]: Css looks good

**Pull request merged**